codecov.yml

```diff
@@ -0,0 +1,22 @@
+# Codecov.io settings 
+
+coverage:
+  status:
+    project:
+      default:
+        # basic settings
+        # auto target means look at deltas from previous
+        target: auto
+        # Threshold we can drop coverage before failing CI
+        threshold: 1%
+        base: auto 
+        flags: 
+          - unit
+        paths: 
+          - "tobac"
+
+# what folders and files to ignore
+ignore:
+  - "tobac/tests/.*"
```

Comment:
Since we defined `omit = tobac/tests/*` in `.coveragerc`, do we actually have to define that again here or how do the `.coveragerc` and the configuration file for `codecov` work together?

Reply:
Would it make sense to also exclude lines that raise `NotImplementedError` and maybe the broken plotting functions discussed #158?

Reply:
You're right that we don't need to ignore/omit the files in both places for codecov.ci. I've kept this as-is though as if developers want to run coverage locally, `.coveragerc` is used, while `codecov.yml` is not. I will add `setup.py` to `.coveragerc` though to keep things consistent. 

I don't believe that we can exclude specific lines of functions. I'm working on a PR though that addresses #158 and will add (limited) tests for those, though.